e2e/tests/home/home-page.spec.ts

```diff
@@ -59,10 +58,10 @@ describe('Home page ui tests', () => {
    // Click Create New Workspace link => Opens Create Workspace page
   test('Click on Create New Workspace link', async () => {
     const home = new HomePage(page);
-    await home.getCreateNewWorkspaceLink().then((link) => link.click());
+    const createLink = await home.getCreateNewWorkspaceLink();
+    await createLink.clickAndWait();
```

Comment:
take into account page navigation happens after click `Create a New Workspace` link in Home page.